orion/pkg/tests_integration/session_integration_test.go

```diff
@@ -0,0 +1,183 @@
+	assert.EqualValues(t, 1, sessions[1].Id)
+	assert.EqualValues(t, "fast_track_2020_spring_class_A", sessions[1].ClassId)
+	assert.EqualValues(t, 2, len(sessions))
+}
+
+// Test: Create 1 Session, Update it, GetBySessionId()
+func Test_UpdateSession(t *testing.T) {
+	resetTable(t, domains.TABLE_SESSIONS)
+	resetTable(t, domains.TABLE_CLASSES)
+	resetTable(t, domains.TABLE_SEMESTERS)
+	resetTable(t, domains.TABLE_LOCATIONS)
+	resetTable(t, domains.TABLE_PROGRAMS)
+
+	// Create 1 Session
+	start := time.Now().UTC()
+	end := start.Add(time.Hour)
+	prog1 := createProgram("fast_track", "Fast Track", 1, 12, "descript1")
+	loc1 := createLocation("loc_1", "4040 Location Rd", "City", "MA", "77294", "Room 1")
+	semester1 := createSemester("2020_spring", "Spring 2020")
+	class1 := createClass("fast_track", "2020_spring", "class_A", "loc_1", "5 pm - 7 pm", start, end)
+	session1 := createSession("fast_track_2020_spring_class_A", start, end, false, "special lecture from guest")
+	body1 := createJsonBody(prog1)
+	body2 := createJsonBody(loc1)
+	body3 := createJsonBody(semester1)
+	body4 := createJsonBody(class1)
+	body5 := createJsonBody(session1)
+	recorder1 := sendHttpRequest(t, http.MethodPost, "/api/programs/v1/create", body1)
+	recorder2 := sendHttpRequest(t, http.MethodPost, "/api/locations/v1/create", body2)
+	recorder3 := sendHttpRequest(t, http.MethodPost, "/api/semesters/v1/create", body3)
+	recorder4 := sendHttpRequest(t, http.MethodPost, "/api/classes/v1/create", body4)
+	recorder5 := sendHttpRequest(t, http.MethodPost, "/api/sessions/v1/create", body5)
+	assert.EqualValues(t, http.StatusOK, recorder1.Code)
+	assert.EqualValues(t, http.StatusOK, recorder2.Code)
+	assert.EqualValues(t, http.StatusOK, recorder3.Code)
+	assert.EqualValues(t, http.StatusOK, recorder4.Code)
+	assert.EqualValues(t, http.StatusOK, recorder5.Code)
+
+	// Update
+	updatedSession := createSession("fast_track_2020_spring_class_A", start, end, true, "cancelled due to corona")
+	updatedBody := createJsonBody(updatedSession)
+	recorder6 := sendHttpRequest(t, http.MethodPost, "/api/sessions/v1/session/1", updatedBody)
+	assert.EqualValues(t, http.StatusOK, recorder6.Code)
+
+	// Get
+	recorder7 := sendHttpRequest(t, http.MethodGet, "/api/sessions/v1/session/1", nil)
+	assert.EqualValues(t, http.StatusOK, recorder7.Code)
+
+	// Validate results
+	var session domains.Session
+	if err := json.Unmarshal(recorder7.Body.Bytes(), &session); err != nil {
+		t.Errorf("unexpected error: %v\n", err)
+	}
+	assert.EqualValues(t, 1, session.Id)
+	assert.EqualValues(t, "fast_track_2020_spring_class_A", session.ClassId)
+	assert.EqualValues(t, "cancelled due to corona", session.Notes)
+}
+
+// Test: Create 1 Session, Delete it, GetBySessionId()
+func Test_DeleteSession(t *testing.T) {
+	resetTable(t, domains.TABLE_SESSIONS)
+	resetTable(t, domains.TABLE_CLASSES)
+	resetTable(t, domains.TABLE_SEMESTERS)
+	resetTable(t, domains.TABLE_LOCATIONS)
+	resetTable(t, domains.TABLE_PROGRAMS)
```

Comment:
Nice! lol consider putting this into a helper function for this class? You repeat this sequence of lines a few times in this file